pkg/test/resourcefixture/testdata/basic/dataflow/v1beta1/dataflowflextemplatejob/streamingdataflowflextemplatejob/_generated_object_streamingdataflowflextemplatejob.golden.yaml

```diff
@@ -3,7 +3,7 @@ kind: DataflowFlexTemplateJob
 metadata:
   annotations:
     cnrm.cloud.google.com/management-conflict-prevention-policy: none
-    cnrm.cloud.google.com/mutable-but-unreadable-fields: '{"spec":{"containerSpecGcsPath":"gs://dataflow-templates/2020-08-31-00_RC00/flex/PubSub_Avro_to_BigQuery","parameters":{"createDisposition":"CREATE_NEVER","inputSubscription":"projects/${projectId}/subscriptions/pubsubsubscription-${uniqueId}","outputTableSpec":"${projectId}:bigquerydataset${uniqueId}.bigquerytable${uniqueId}","outputTopic":"projects/${projectId}/topics/pubsubtopic1-${uniqueId}","schemaPath":"gs://config-connector-samples/dataflowflextemplate/numbers.avsc"}}}'
+    cnrm.cloud.google.com/mutable-but-unreadable-fields: '{"spec":{"containerSpecGcsPath":"gs://dataflow-templates/2020-08-31-00_RC00/flex/PubSub_Avro_to_BigQuery","parameters":{"createDisposition":"CREATE_NEVER","inputSubscription":"projects/${projectId}/subscriptions/pubsubsubscription-${uniqueId}","outputTableSpec":"${projectId}:bigquerydataset${uniqueId}.bigquerytable${uniqueId}","outputTopic":"projects/${projectId}/topics/pubsubtopic0-${uniqueId}","schemaPath":"gs://config-connector-samples/dataflowflextemplate/numbers.avsc"}}}'
```

Comment:
May I ask why this is a mutable but unreadable fields?
I'm thinking that since we are migrating to the direct resource, we should drop this annotation and switch to some other mechanism (I don't think the mutable-but-unreadable-fields in general will break users).

Reply:
Ah sure - this test is actually _not_ using the direct controller.  This is actually me noticing another problem in our tests and fixing it.  I will just split out this PR, it's not really relevant here!

We do use the direct controller in  `batchdataflowflextemplatejob-direct`, but that is actually using the annotation to opt-in to the direct controller.

Reply:
Ah, gotcha.